<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Content</class>
 <widget class="QWidget" name="Content">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolCursor">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/cursor.ico</normaloff>:/icons/icons/cursor.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolMove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/move.ico</normaloff>:/icons/icons/move.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolRotate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/rotate.ico</normaloff>:/icons/icons/rotate.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>8</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777214</width>
         <height>8</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SwitchToolButton" name="toolModel">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/cube.ico</normaloff>:/icons/icons/cube.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>38</x>
          <y>34</y>
          <width>12</width>
          <height>12</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>▶</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="SwitchToolButton" name="toolLight">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/sun.ico</normaloff>:/icons/icons/sun.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>38</x>
          <y>34</y>
          <width>12</width>
          <height>12</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>▶</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PreviewWidget" name="openGLWidget"/>
   </item>
   <item>
    <widget class="NavDockWidget" name="modelWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>136</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>524287</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="windowTitle">
      <string>模型列表</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <item>
        <widget class="ColumnList" name="modelList">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ModelPropWidget" name="modelProp" native="true">
         <property name="minimumSize">
          <size>
           <width>282</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="NavDockWidget" name="lightWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>136</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>524287</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="windowTitle">
      <string>光源列表</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <item>
        <widget class="ColumnList" name="lightList">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LightPropWidget" name="lightProp" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>previewwidget.h</header>
  </customwidget>
  <customwidget>
   <class>SwitchToolButton</class>
   <extends>QToolButton</extends>
   <header>controls.h</header>
  </customwidget>
  <customwidget>
   <class>NavDockWidget</class>
   <extends>QDockWidget</extends>
   <header>controls.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModelPropWidget</class>
   <extends>QWidget</extends>
   <header>propertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LightPropWidget</class>
   <extends>QWidget</extends>
   <header>propertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColumnList</class>
   <extends>QTableView</extends>
   <header>controls.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
